packages/zoe/src/zoeService/zoe.js

```diff
@@ -35,15 +32,19 @@ import { setupCreateZCFVat } from './createZCFVat';
  * @param {string} [zcfBundleName] - The name of the contract facet bundle.
  * @returns {ZoeService} The created Zoe service.
  */
-function makeZoe(vatAdminSvc, zcfBundleName = undefined) {
-  /** @type {WeakStore<SeatHandle, ZoeSeatAdmin>} */
-  const seatHandleToZoeSeatAdmin = makeNonVOWeakStore('seatHandle');
+const makeZoe = (vatAdminSvc, zcfBundleName = undefined) => {
+  // We must use the ZoeService before it is defined. See below where
+  // the promise is resolved.
```

Comment:
Using it before it is defined is strange. Do you mean you send it eventual messages before it is defined, to be delivered once it is defined?

Reply:
Fixed comment